examples/select-combobox-focus-within/test.ts

```diff
@@ -1,16 +1,16 @@
 import { click, press, q, type } from "@ariakit/test";
 
-function setup() {
+beforeEach(() => {
```

Comment:
If I recall correctly, we used these `setup` functions instead of hooks because otherwise they would run before the component was mounted by the Vitest setup. This caused the `External button` element to be appended to the body before the component under test, potentially affecting the results. It's odd that nothing has broken now, though I'm not sure if this is a remnant from before migrating from Jest to Vitest.

Reply:
@diegohaz 
I think nothing is broken because of the work @DaniGuardiola did making tests framework agnostic. The result is that mounting happens during a `beforeEach`, which happens before the hooks in the component.

I was curious why the setup functions weren't in the examples but I guess that doesn't matter now!

Reply:
That's great! Those setup functions were only added to include before and after tabbable elements, allowing us to test things like tab order. Since the browser already has its chrome, we don't need them in the examples themselves.